configure.ac

```diff
@@ -483,6 +477,10 @@ AC_SUBST([PART_LDFLAGS])
 CFLAGS=$SAVE_CFLAGS
 LDFLAGS=$SAVE_LDFLAGS
 
+
+# For disk notification and fault LED.
+PKG_CHECK_MODULES([LIBLSM], [libstoragemgmt >= 1.3.0])
```

Comment:
Oh, I see it now there, thanks! I also have an update on this - I just talked to @tsmetana about this and he agrees that we should avoid adding a hard dependency on *LSM*. It should be only the module having the dependency so that distributions may easily get along without *LSM*. So these new interfaces should be added by the LSM module (for existing block device objects).